functests/utils/k8sreporter/example/main.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
done we need this file?

Reply:
I would say yes, for multiple reasons.
- My idea is that at some point we can even ship this as a library that consumed by others (I put it here mostly for delivery speed reasons)
- It provides a nice example of how to use / configure the reporter
- It's an easy way to check / debug the reporter without having to run the whole suite